ClientApp/src/Pages/GettingStarted/GettingStarted.tsx

```diff
@@ -0,0 +1,104 @@
+
+import React, { useState } from 'react';
+import LocationData from './LocationData.json';
+import './GettingStarted.scss';
+
+import { LocationSummary } from './LocationSummary';
+import ReactWeather, { useOpenWeather } from 'react-open-weather';//ignore red-line, it still works
+
+enum Locations {
+    Scotland,
+    Canada,
+    Australia
+}
+
+function GettingStarted() {
+    const [selectedLocation, setSelectedLocation] = useState(Locations.Scotland);
+
+    let x;
+    let y;
+    let loc;
+
+    const latScotland = '57.7207794189';
+    const LonScotland = '-5.6862931252';
+    const LatCanada = '53.135509';
+    const LonCanada = '-57.660435';
+    const LatAustralia = '-20.917574';
+    const LonAustralia = '142.702789';
+
+    if (selectedLocation === Locations.Canada) {
+        x = LatCanada;
+        y = LonCanada;
+        loc = " NewFoundland, Canada";
+    }
+    else if (selectedLocation === Locations.Scotland) {
+        x = latScotland;
+        y = LonScotland;
+        loc = "Gairloch, Scotland"
+    }
+    else if (selectedLocation === Locations.Australia) {
+        x = LatAustralia;
+        y = LonAustralia;
+        loc = "Queensland, Australia"
+    }
```

Comment:
this works totally fine, but I think we could make it a little bit cleaner with a little refactor.
Let's pull this whole thing out into its own little function, and then our component can use the data that its given.

ie
```ts
interface LocationData {
    latitude: number;
    longitude: number;
    label: string;
}

const canadaLocationData = {
    latitude: 53.135509,
    longitude: -57.660435,
    label: "NewFoundland, Canada",
};

const scotlandLocationData = { /* stuff */ };
const austrailiaLocationData = { /* stuff */ };

function getLocationData(location: Locations): LocationData {
    switch (location) {
        case: Locations.Canada:
            return canadaLocationData;
        case: Locations.Scotland:
            return scotlandLocationData;
        case: Locations.Australia:
            return australiaLocationData;
    }
}
```

Then out component can just do

```ts
const locationData = getLocationData();

const { data, isLoading, errorMessage } = useOpenWeather({
    key: 'stuff',
    lat: locationData.latitude,
    lon: locationData.longitude,
    lang: 'en',
    unit: 'metric'
});
```